mods/default/depends.txt

```diff
@@ -0,0 +1 @@
+players?
```

Comment:
This should be hard dependency, if players mod isn't installed, there will be an error in `legacy.lua` because there the `players` table is used and it mustn't be `nil`.